spec/spec_helper.rb

```diff
@@ -1 +1,67 @@
+RSpec.configure do |c|
+  c.mock_with :rspec
+end
+
 require 'puppetlabs_spec_helper/module_spec_helper'
+
+fixture_path = File.expand_path(File.join(__FILE__, '..', 'fixtures'))
+module_name = File.basename(File.expand_path(File.join(__FILE__,'../..')))
+
+default_hiera_config =<<-EOM
+---
+
+defaults:
+  datadir: data
+  data_hash: yaml_data
+
+hierarchy:
+  - name: 'OS Major Version'
+    path: '%{facts.os.family}.%{facts.os.name}.%{facts.os.release.major}.yaml'
+  - name: 'OS Name'
+    path: '%{facts.os.family}.%{facts.os.name}.yaml'
+  - name: 'OS Family'
+    path: '%{facts.os.family}.yaml'
+
+version: 5
```

Comment:
I believe this is supposed to be at the top of the `hiera.yaml` content.

Reply:
I obsessively sort my yaml keys.  It's okay at the bottom.